app/helpers/user_helper.rb

```diff
@@ -21,6 +21,10 @@ def moderation_view?
     current_user&.mod? && session[:moderation_view] == true
   end
 
+  def belongs_to_current_user?
```

Comment:
This method would be better suited as a private method inside UserController; helpers should be for views only.

btw, you can also use Ruby3's endless method definition syntax:
```ruby
private

def belongs_to_current_user? = @user == current_user
```